source/includes/fundamentals/code-examples/CountDocuments.cs

```diff
@@ -0,0 +1,62 @@
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+namespace TestRun.Fundamentals;
+public class CountDocuments
+{
+    private static IMongoCollection<Student> _myColl;
+    private const string MongoConnectionString = "<Your MongoDB URI>";
+    public static void Main(string[] args)
+    {
+        Setup();
+        InsertSampleData();
+        
+        // start-accurate-ct
+        var filter = Builders<Student>.Filter.Lt(s => s.FinalGrade, 80.0);
+        var count1 = _myColl.CountDocuments(filter);
+        Console.WriteLine("Number of documents with a final grade less than 80: " + count1);
+        // end-accurate-ct
+
+        // start-est-count
+        var count2 = _myColl.EstimatedDocumentCount();
+        Console.WriteLine("Estimated number of documents in the students collection: " + count2);
+        // end-est-count
+
+        // start-agg-count
+        var matchStage = Builders<Student>
+            .Filter.Gt(s => s.FinalGrade, 80);
+        var result = _myColl.Aggregate().Match(matchStage).Count();
+        Console.WriteLine("Number of documents with a final grade more than 80: "+result.First().Count);
+        // end-agg-count
+        
+        _myColl.DeleteMany(Builders<Student>.Filter.Empty);
+    }
+    private static void InsertSampleData()
+    {
+        var studentList = new List<Student>()
+        {
+            new() { Id= 1, Name = "Jonathon Howard ", FinalGrade = 87.5 },
+            new() { Id= 2, Name = "ABCDEF Howard ", FinalGrade = 10.5 },
+            new() { Id= 3, Name = "aeufh Howard ", FinalGrade = 90.5 }
```

Comment:
I: since this is a public repo, make sure that the sample code matches what's in the page. 
S: replace with the same sample data as in the page.

Reply:
Bumping this comment since I dont think you saw it!